src/main/java/com/google/devtools/build/lib/exec/SpawnInputExpander.java

```diff
@@ -222,7 +223,8 @@ private static void addInputs(
       NestedSet<? extends ActionInput> inputFiles,
       ArtifactExpander artifactExpander,
       PathFragment baseDirectory) {
-    List<ActionInput> inputs = ActionInputHelper.expandArtifacts(inputFiles, artifactExpander);
+    List<ActionInput> inputs = ActionInputHelper.expandArtifacts(inputFiles, artifactExpander,
```

Comment:
The returned collection of `SpawnInputExpander#getInputMapping` was expected to not contain tree artifacts. but now it is changed. Can you add a comment to the function's javadoc?

Reply:
Good point, I augmented the comment. I tried to distinguish between "empty TreeArtifact" and "TreeArtifact expanding to the empty set with the given ArtifactExpander", as the coverage dir issue shows these are different concepts. Let me know if this is overly verbose.